lib/common.js

```diff
@@ -187,6 +187,7 @@ function deepextend() {
   return out
 }
 exports.deepextend = deepextend
+exports.deep = deepextend
```

Comment:
backwards / forwards compat with something?

Reply:
yes - this is an alias for the current name in the latest Seneca